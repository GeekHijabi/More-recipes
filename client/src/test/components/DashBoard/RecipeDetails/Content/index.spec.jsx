import React from 'react';
import { shallow } from 'enzyme';
import { Content }
  from '../../../../../js/components/DashBoard/RecipeDetails/Content';

const props = {
  recipe: {
    ingredient: '',
    description: '',
  },
  description: 'boil rice',
  activeTab: {
    ingredients: 'ingredients',
    description: 'description',
    reviews: 'reviews'
  },
  reviewedRecipe: {
    id: 1,
    recipeName: 'rice',
    description: 'boil rice',
    ingredients: 'rice, water'
  },
  recipeId: 2
};


describe('Content snapshot', () => {
  it('should render correctly', () => {
    const content = shallow(<Content {...props} />);
    expect(content).toMatchSnapshot();
  });
});
