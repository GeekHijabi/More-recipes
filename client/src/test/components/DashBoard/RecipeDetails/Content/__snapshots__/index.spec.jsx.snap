// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content snapshot should render correctly 1`] = `
<div
  className="tab-content"
  id="myTabContent"
>
  <Description
    activeTab={
      Object {
        "description": "description",
        "ingredients": "ingredients",
        "reviews": "reviews",
      }
    }
    description=""
  />
  <Ingredients
    activeTab={
      Object {
        "description": "description",
        "ingredients": "ingredients",
        "reviews": "reviews",
      }
    }
    ingredients=""
  />
  <Connect(Reviews)
    activeTab={
      Object {
        "description": "description",
        "ingredients": "ingredients",
        "reviews": "reviews",
      }
    }
    recipeId={2}
    reviewedRecipe={
      Object {
        "description": "boil rice",
        "id": 1,
        "ingredients": "rice, water",
        "recipeName": "rice",
      }
    }
  />
</div>
`;
